boards/native/Makefile.dep

```diff
@@ -16,6 +16,7 @@ endif
 ifneq (,$(filter socket_zep,$(USEMODULE)))
   USEMODULE += iolist
   USEMODULE += netdev_ieee802154
+  USEMODULE += netif
```

Comment:
```suggestion
  USEMODULE += netif_str
```
?

otherwise this has the potential to introduce cyclic dependencies, as this goes "up" for some reason...

Reply:
Should that be a submodule of `netif`?
An entirely separate module that lives in `sys/net/netif_str`?
Should the header be separated as well?

Reply:
If you use `SUBMODULE` for submodule you would need to have `netif` as a dependency.... :-/ How about we add this functions to `l2util`?

Reply:
We should get this going... See https://github.com/RIOT-OS/RIOT/pull/15310#pullrequestreview-522622761